### DISCUSSION TITLE
Customize how many words do I see per line

### DISCUSSION THREAD

**Discussion by overthetop** [monkeytypegame/monkeytype#1554] [Questions] [answered]
Is there a way to set the words per line that are visible? For example, when I start to practice, I'd like to see only 4 words per line. Is that possible somehow?

**Comment by Miodec** [comment, marked as answer]
Currently there is no way to say exactly how many words you want to see per line, but you can mess around with `page width` and `font size` settings to kind of do that.

**Reply by overthetop** [reply]
Any chance that could become a feature?

**Reply by ShizukoV** [reply]
maybe but maybe not lol ;)

**Reply by Miodec** [reply]
Honestly i dont see much point in that

**Reply by overthetop** [reply]
@Miodec that will help focus better when you're reading ahead

**Comment by overthetop** [comment]
@Miodec can you please help me accomplish the look in that video https://www.youtube.com/watch?v=YOhi8MqKnRc ? How to configure to show only 3 words ahead continuously?

**Reply by Miodec** [reply]
Use the plus two funbox

**Reply by overthetop** [reply]
@Miodec can you please guide me a little bit more how to do that exactly? Should I apply a style to an html element in the chrome console or do something else?

**Reply by overthetop** [reply]
Ohh I've found it. It's in the monkeytype settigns. One more thing. How to configure the words per line (or the font) just like in the video?

**Reply by Miodec** [reply]
You cant change the words per line, but you can change the page width and font size. Those settings and the font family are all available in the settings page.

**Reply by overthetop** [reply]
Thank you @Miodec Increasing the font size did it
